Submissions/NOFAXKING.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "[TEMGOUA TEMGOUA Arthur]",
```

```suggestion
    "name": "TEMGOUA TEMGOUA Arthur",
```

Please remove the placeholder brackets, not only in this example, but in the complete file.